scripts/analysis/amwg_table.py

```diff
@@ -1,3 +1,23 @@
+import numpy as np
+import xarray as xr
+from pathlib import Path
+import warnings  # use to warn user about missing files.
+
+try:
+    import scipy.stats as stats # for easy linear regression and testing
+except ImportError:
+    print("Scipy module does not exist in python path, but is needed for amwg_table.")
+    print("Please install module, e.g. 'pip install scipy'.")
+
+#Import "special" modules:
```

Comment:
I would put this comment above the `scipy.stats` import, as it is also a "special" module.